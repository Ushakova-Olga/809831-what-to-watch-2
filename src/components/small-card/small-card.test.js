import React from "react";
import renderer from 'react-test-renderer';
import SmallCard from '../small-card/small-card';

it(`Small card correctly renders`, () => {
  const testObject = {name: `Fantastic Beasts`, img: `bohemian-rhapsody.jpg`};
  const tree = renderer
    .create(<SmallCard information={testObject} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
