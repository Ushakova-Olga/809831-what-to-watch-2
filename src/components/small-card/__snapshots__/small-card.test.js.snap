// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Small card correctly renders 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/films/0"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="100%"
        muted={true}
        poster="fantastic-beasts-the-crimes-of-grindelwald.jpg"
        width="100%"
      >
        <source
          src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
          type="video/mp4"
        />
      </video>
    </div>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/0"
      onClick={[Function]}
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
